config/routes.rb

```diff
@@ -2,7 +2,7 @@
   root 'parks#index'
   devise_for :users
   resources :parks, only: [:index, :new, :create, :show, :edit, :update, :destroy]do
-    resources :reviews, only: [:new, :create]
+    resources :reviews, only: [:new, :create, :edit, :update, :destroy]
   end
 
 
```

Comment:
I can't leave a comment on the file below 👇,  but it looks like you accidentally added a file

Reply:
The file name is `dog-park-project`